src/cli/gentest/test_providers.py

```diff
@@ -20,9 +20,9 @@
 from .request_manager import RPCRequest
 
 
-class BlockchainTestProvider(BaseModel):
+class TransactionTestProvider(BaseModel):
```

Comment:
We might need to rename this - we'll likely add the transaction test format from ethereum/tests soonish, here's a [link to the ethereum/tests docs](https://ethereum-tests.readthedocs.io/en/latest/test_types/transaction_tests.html). So I think we have to call it as following to avoid any ambiguity later.
```suggestion
class StateTestProvider(BaseModel):
```